Sources/DolbyIORTSUIKit/Private/Views/StatsInfo/StatsInfoButton.swift

```diff
@@ -7,18 +7,18 @@ import DolbyIOUIKit
 import SwiftUI
 
 struct StatsInfoButton: View {
+    
+    private let onAction: () -> Void
 
-    @StateObject private var viewModel: StatsInfoViewModel
-
-    init(streamSource: StreamSource) {
-        _viewModel = StateObject(wrappedValue: StatsInfoViewModel(streamSource: streamSource))
+    init(onAction: @escaping (() -> Void) = {}) {
```

Comment:
Since the StatsInfoButton need an action every time its used - lets not assign default values to the argument ?